docker/docker-compose.yml

```diff
@@ -36,8 +36,8 @@ services:
     environment:
       UWSGI_PROCESSES: 6
       UWSGI_THREADS: 2
-      SIMPLIFIED_PRODUCTION_DATABASE: postgres://simplified:password@db:5432/simplified_circulation_dev
-      SIMPLIFIED_TEST_DATABASE: postgres://simplified_test:password@db:5432/simplified_circulation_test
+      SIMPLIFIED_PRODUCTION_DATABASE: postgresql://simplified:password@db:5432/simplified_circulation_dev
+      SIMPLIFIED_TEST_DATABASE: postgresql://simplified_test:password@db:5432/simplified_circulation_test
```

Comment:
It occurs to me we're going to have to mirror this change in our build processes, since that's where the environment variables come from.